agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -244,6 +250,38 @@ public AzureMonitorExporterBuilder credential(TokenCredential credential) {
    *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
    */
   public AzureMonitorTraceExporter buildTraceExporter() {
+    initExporterBuilder();
+    SpanDataMapper mapper =
+        new SpanDataMapper(
+            true,
+            builder -> {
+              builder.setInstrumentationKey(instrumentationKey);
+              builder.addTag(
+                  ContextTagKeys.AI_INTERNAL_SDK_VERSION.toString(),
+                  VersionGenerator.getSdkVersion());
+            },
+            (event, instrumentationName) -> false,
+            () -> null);
+
+    return new AzureMonitorTraceExporter(mapper, telemetryItemExporter);
+  }
+
+  /**
+   * Creates an {@link AzureMonitorMetricExporter} based on the options set in the builder. This
+   * exporter is an implementation of OpenTelemetry {@link MetricExporter}.
+   *
+   * @return An instance of {@link AzureMonitorMetricExporter}.
+   * @throws NullPointerException if the connection string is not set on this builder or if the
+   *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
+   */
+  public AzureMonitorMetricExporter buildMetricExporter(
+      Consumer<AbstractTelemetryBuilder> telemetryInitializer) {
```

Comment:
let's not expose `telemetryInitializer` in the public API, we probably want to steer standalone users to a more "OpenTelemetry"-way like using a SpanProcessor for this kind of functionality

Reply:
updated.